Blog Post: Long-Term Support for Stuart Bowman (#247)
OpenSats/website: data/blog/stuart-bowman-receives-lts-grant.mdx

---
title: "Long-Term Support For Stuart Bowman"
date: '2024-04-09'
tags: ['OpenSats', 'grants', 'nostr', 'LTS']
draft: false
authors: ['arvin', 'dergigi']
images: ['/static/images/blog/28-stuart-lts.jpg']
summary: "OpenSats Welcomes Stuart Bowman as an LTS Grantee."
---

We are happy to announce that another nostr developer is joining our roster of
LTS Grantees: [Stuart Bowman](https://github.com/lovvtide).

Stuart is the main developer and maintainer of [Satellite
Earth](https://satellite.earth/), a web-based nostr client and CDN server that
is geared towards self-sovereign communities.

The overarching goal of Satellite is to strengthen the nostr network by
strengthening the various communities that make up the network. The “Satellite
Stack” will allow communities to be self-reliant and thus robust, improving the
relay system for greater diversity and functionality in the process. In the end,
this will expand the network's capabilities and increase its value and activity.

Stuart will continue to work on the Satellite Stack and related components, such
as:

> - [Satellite Node](https://github.com/lovvtide/satellite-node): A nostr node with enhanced relay capabilities, providing a dedicated backend for communities.

Reviewer: ```suggestion
- [Satellite Public Node](https://github.com/satellite-earth/public-node): A "nostr node" with enhanced relay capabilities, providing a dedicated backend for communities.
```

> - [Satellite Control](https://github.com/lovvtide/satellite-control): A control panel that allows non-technical users to operate their relay and media archive.

Reviewer: ```suggestion
- [Satellite Control](https://github.com/satellite-earth/dashboard-ui): A control panel that allows non-technical users to operate their relay and media archive.
```

> - [Satellite Desktop](https://github.com/lovvtide/satellite-desktop): A cross-platform desktop app to easily run satellite nodes on consumer hardware.

Reviewer: ```suggestion
- [Satellite Desktop](https://github.com/satellite-earth/desktop): A cross-platform desktop app to easily run satellite nodes on consumer hardware.
```

> - [Satellite CDN](https://github.com/lovvtide/satellite-cdn): A Blossom-compatible CDN server that handles media uploads, downloads, user authentication, and file indexing.

Reviewer: ```suggestion
- [Satellite CDN](https://github.com/lovvtide/satellite-cdn): A Blossom-compatible CDN server that handles media uploads, downloads, user authentication, lightning payments, and file indexing.
```

> 

Reviewer: ```suggestion
- [Satellite Private Node](https://github.com/satellite-earth/private-node): A local nostr relay and media proxy to boost speed, privacy, and network redundancy.

```
